h/presenters.py

```diff
@@ -128,14 +125,13 @@ def title(self):
 
         """
         document_ = self.annotation.document
```

Comment:
`annotation.document` can be `None` with the new PG setup. We should probably add a check for this.